src/ONYXKEYS.js

```diff
@@ -71,6 +71,9 @@ export default {
     // Contains all the users settings for the Settings page and sub pages
     USER: 'user',
 
+    // Contains user login list
+    USER_LOGIN_LIST: 'userLoginList',
```

Comment:
Could this just be `loginList` instead of `userLoginList`? `loginList` matches the name we use in the backend for this data.